### ISSUE TITLE
Each disease page has distorted UI

### ISSUE THREAD

**Issue by akshitadixit** [GDSC-IIIT-Kalyani/MediCare-Prime#4]
**Describe the bug**
Every disease page has the entire page collapsing onto the navigation bar and looks ugly

**To Reproduce**
1. Go to https://medicare-prime.herokuapp.com
2. Click on 'Departments'
3. Click on any department
4. See the trouble happening

**Expected behavior**
The rest of the page lies below the navbar, cleanly(?)

**Screenshots**
![Screenshot from 2021-10-20 16-01-35](https://user-images.githubusercontent.com/56997545/138077375-a59ee002-e0f0-4af7-8cef-71c1f52ab748.png)

**Comment by VIKAS0264** [comment]
Can I work on this ?

**Comment by Isha307** [comment]
@ghousahmed Are you still working on it?

**Comment by 01-DC** [comment]
I think this UI issue has been fixed. However I had another suggestion for disease page.

**Comment by Isha307** [comment]
Yes, but still there are so many problem in the predictor pages we can fix that.

**Comment by 01-DC** [comment]
Yes, I have already taken note of the issues in the prediction result pages. Any other issues which you would point out? @Isha307

**Comment by Isha307** [comment]
At present, work on it. Later we'll see if we find some bug.
